backend/tracim_backend/tests/functional/test_message.py

```diff
@@ -586,6 +586,41 @@ def test_api__read_all_messages__ok_204__nominal_case(self, session, web_testapp
         ).json_body.get("items")
         assert len(message_dicts) == 3
 
+    def test_api__read_content_related_messages__ok_204__nominal_case(
+        self, session, web_testapp
+    ) -> None:
+        """
+        Read all unread messages
+        """
+        create_content_messages(session, unread=True, sent_date=datetime.datetime.utcnow())
+
+        web_testapp.authorization = ("Basic", ("admin@admin.admin", "admin@admin.admin"))
+
+        message_dicts = web_testapp.get(
+            "/api/users/1/messages?read_status=unread", status=200,
+        ).json_body.get("items")
+        assert len(message_dicts) == 4
+
+        web_testapp.put("/api/users/1/messages/read?content_ids=1", status=204)
+
+        message_dicts = web_testapp.get(
+            "/api/users/1/messages?read_status=unread", status=200,
+        ).json_body.get("items")
+        assert len(message_dicts) == 3
+
+        web_testapp.put("/api/users/1/messages/read?parent_ids=1", status=204)
+
+        message_dicts = web_testapp.get(
+            "/api/users/1/messages?read_status=unread", status=200,
+        ).json_body.get("items")
+        assert len(message_dicts) == 2
+
+        web_testapp.put("/api/users/1/messages/read?content_ids=3,4", status=204)
+        message_dicts = web_testapp.get(
+            "/api/users/1/messages?read_status=unread", status=200,
+        ).json_body.get("items")
+        assert len(message_dicts) == 0
```

Comment:
It would be great if you had a message unrelated to contents 3 and 4 that is sill unread in this test, to show that the API does not erroneously mark all messages as read.

Reply:
it's already verified before, this is just few brief tests cases. Test doesn't also verify if read message are the correct one.

Reply:
Ah yes, my bad, it's okay then